src/Polytopes/Polyhedron/properties.jl

```diff
@@ -89,19 +99,15 @@ function faces(as::Type{T}, P::Polyhedron, face_dim::Int) where {T}
     if as == Polyhedron || as == Polyhedra
         PolyhedronFacePolyhedronIterator(P,face_dim-size(lineality_space(P),1))
     else
-        throw(ArgumentError("Unsupported `as` argument :" * string(as)))
+        throw(ArgumentError("Unsupported `as` argument:" * string(as)))
```

Comment:
String interpolation would be nicer:
```suggestion
        throw(ArgumentError("Unsupported `as` argument $(as)"))
```
Also, this line is not tested; I recommend to also test error handling code like this whenever possible with reasonable effort